content/300-guides/050-database/820-multi-schema.mdx

```diff
@@ -9,13 +9,13 @@ tocDepth: 2
 
 <Admonition type="warning">
 
-Multiple database schema support is currently only available with the PostgreSQL connector.
+Multiple database schema support is currently available with the PostgreSQL and CockroachDB connectors.
```

Comment:
Since we are only adding support for the multiSchema preview feature to Cockroachdb in this release, does it make sense to add "since 4.8.0"?

Reply:
I'm unsure, but open to thoughts on this. Since multi-schema is getting updated over so many releases, it's going to get cluttered explaining exactly what went in in each version. PostgreSQL had an early 'experimental' release in 4.3.0 and then a proper release in 4.7.0, do we need to mention both?

On the other hand, if the information is useful maybe it should be there.

Reply:
Let's just leave it as is. If we try to reflect the reality per version per database connector its going to be super messy.